cores/arduino/startup.c

```diff
@@ -37,7 +37,7 @@
 //USE DPLL0 for 120MHZ
 #define MAIN_CLOCK_SOURCE				  GCLK_GENCTRL_SRC_DPLL0
 
-#define GENERIC_CLOCK_GENERATOR_1M		  (5u)
+#define GENERIC_CLOCK_GENERATOR_1M		  (7u)
```

Comment:
is GCLK 7 available on the SAMDG19?

Reply:
yup clock unit is the same on all. Tested w/ itsybitsy m4 to be sure.